lib/bots/versionbot.ts

```diff
@@ -1429,13 +1488,22 @@ export class VersionBot extends ProcBot {
 			_.each(protectedContexts, (proContext) => {
 				// We go through every status and see if the context prefixes the context
 				// of the status.
+				// We filter the VersionBot reviews.
```

Comment:
nit: there appears to be nothing in the filtering below hardcoded to `VersionBot` ?